models/action.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose')
+
+const ActionSchema = new mongoose.Schema({
+  name: { type: String, required: true },
+  description: { type: String },
+  actorId: [{ type: mongoose.Schema.Types.ObjectId, ref: 'Actor' }]
+}, {
+  strict: false
+})
```

Comment:
Add [Cascade delete](https://stackoverflow.com/questions/14348516/cascade-style-delete-in-mongoose)?

Reply:
La cascade devrait pultôt être sur l'acteur, mais l'acteur ne référence pas l'actions (bonnes pratiques mongoose AFAIK). Je crée une issue pour faire un trigger.